### ISSUE TITLE
KiTTY menues aren't showed in tab session

### ISSUE THREAD

**Issue by crisistotal2014** [jimradford/superputty#605]
Hi!
I prefer to use KiTTY rather than PuTTY (basically due file transfer easy actions).
But in SuperPutty KiTTY menues aren't showed in tab session (in "MTPutty" they are showed).

It's possible to show in tab session? (attached an screenshot of KiTTY menu)
![kitty tab menu](https://cloud.githubusercontent.com/assets/18112277/14074077/9e3f72d2-f4ce-11e5-9cd1-eeac0462bff8.png)

**Comment by sethkush** [comment]
It's not a great fix, but ExtraPutty's menus work fine. You could use ExtraPutty. http://extraputty.com/

**Comment by crisistotal2014** [comment]
Yeah, but I want to use SuperPutty with KiTTY as your programs seems to publish!!

**Comment by sethkush** [comment]
Publish?

**Comment by klonos** [comment]
I guess he means advertise/claim to support.

**Comment by crisistotal2014** [comment]
Exact! Thanks klonos!
It's possible to implement like "MTPuTTY"?

**Comment by kabracity** [comment]
+1 to the request :)
